server/routing/lobby.go

```diff
@@ -71,6 +72,45 @@ func (lr *LobbyRoutes) lobbyJoinHandler(w http.ResponseWriter, r *http.Request)
 	return
 }
 
+func (lr *LobbyRoutes) lobbyReadyHandler(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	lobbyID := vars["lobbyID"]
+
+	playerName, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		panic(err)
+	}
+
+	lobby, err := lr.Services.Lobby.ReadyPlayer(lobbyID, string(playerName))
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusNotFound)
+		return
+	}
+
+	msg := pogo.ReadyMsg{
+		ID:      lobby.ID.String(),
+		Players: lobby.Players,
+		ReadyStatus: string(playerName) +" is "+ lobby.PlayerReady[string(playerName)],
+	}
+
+	bytes, err := json.Marshal(msg)
+	if err != nil {
+		http.Error(w, "failed to build lobby message", http.StatusInternalServerError)
+		return
+	}
+
+	w.Write([]byte(bytes))
```

Comment:
Similar to my comments on #23, I think the response from the handler should be just a simple `200` saying "marked as ready!" and the later pubsub interaction will push out the actual lobby update message.